executor/simple.go

```diff
@@ -1067,21 +1074,83 @@ func (e *SimpleExec) executeSetPwd(s *ast.SetPwdStmt) error {
 	return err
 }
 
-func (e *SimpleExec) executeKillStmt(s *ast.KillStmt) error {
-	conf := config.GetGlobalConfig()
-	if s.TiDBExtension || conf.CompatibleKillQuery {
```

Comment:
`conf.CompatibleKillQuery` can be removed from config.go and config.toml.example.

Reply:
And a following PR for parser can be filed to change the [comment of `TiDBExtension`](https://github.com/pingcap/parser/blob/master/ast/misc.go#L755-L762)